src/exclusions.js

```diff
@@ -0,0 +1,10 @@
+const { combinationOfTraitsAlreadyExists } = require('./exclusions/combination_traits');
+const { incompatibleTraitsUsed } = require('./exclusions/incompatible_traits');
+
+const needsExclusion = (selectedTraitsList, newTraits, maxRepeatedTraits, incompatibleTraits) => {
```

Comment:
this method combines the two new exclusion functionalities, other exclusion patterns could be easily added here